src/index.ejs

```diff
@@ -76,7 +76,11 @@
             {
               type:'welcome',
               options:{
-                title:'Open your new bank account'
+                title:'Open your new bank account',
+                descriptions: [
+                  'To open a bank account, we will need to verify your identity.',
```

Comment:
It is in the Read me ;) https://github.com/onfido/onfido-sdk-ui/tree/development#welcome